package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@decisively-io/interview-react-material",
-  "version": "0.6.9",
+  "version": "0.6.9-86b2a2535-4",
```

Comment:
@isRostCompany just make sure these are the proper version once the SDK is published